lib/ansible/modules/system/filesystem.py

```diff
@@ -135,6 +144,17 @@ def create(self, opts, dev):
             cmd = "%s %s %s '%s'" % (mkfs, self.MKFS_FORCE_FLAGS, opts, dev)
         self.module.run_command(cmd, check_rc=True)
 
+    def get_uuid(self, dev):
+        """ Fetch the UUID """
+        blkid = self.module.get_bin_path('blkid', required=True)
+        cmd = "{0} {1} -s UUID -o value".format(blkid, dev)
+        _, out, _ = self.module.run_command(cmd, check_rc=True)
```

Comment:
`check_rc` is `False` when used [blkid is previously execued](https://github.com/ansible/ansible/blob/8534178b1d28f2134da8f6b6634879157b5e34e9/lib/ansible/modules/system/filesystem.py#L398). Should not both `check_rc` parameter be equal ?